### PR TITLE
 Display list of items on the Homepage - Student A

### PR DESCRIPTION
**Pull request by SebastianSolisFenger**
Create the main part of the homepage that keeps the layout from the wireframe:

### CODE REVIEW

**Comment by alexpuente0** [comment]
I'll review your code right away!

**Comment by alexpuente0** [comment]
Excellent choice with the NASA API and the background image choice! 🌑 

Great use of modules, code is very well organized and readable.
I see no conflict and all linters are cleared, this branch is ready for merge.

Cheers!

**Review by alexpuente0 — APPROVED** [review]
Excellent choice with the NASA API and the background image choice! 🌑

Great use of modules, code is very well organized and readable.
I see no conflict and all linters are cleared, this branch is ready for merge.

Cheers! ⭐ ⭐ ⭐

File: src/modules/API-links.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  '2022-04-14',
+  '2022-04-15',
+  '2022-04-16',
+  '2022-04-17',
+  '2022-04-18',
+];
+
+const getDataDateImage = (num) => `https://api.nasa.gov/planetary/apod?api_key=yE5XwF3YBRu6RaMb2K328lXJabWCog5rzjaIR76N&date=${randomDatesForPopupImgs[num]}`;
+
+export { getNasaApi, getDataDateImage };
```
[the remark this answers is not in the document]
Comment (SebastianSolisFenger) [reply src/modules/API-links.js:17]: Thanks!

**Pull request merged**